### DISCUSSION TITLE
Como centralizar imagens

### DISCUSSION THREAD

**Discussion by PsyDrops** [PsyDrops/PsyDrops#2] [Q&A] [answered]
Qual técnica devo utilizar para centralizar imagens nos meus Readme?

**Comment by wesleyallan** [comment, marked as answer]
Uma forma simples e rápida de resolver esse problema é adicionar um elemento `<div>` como pai da imagem adicionada via Markdown e adicionar a propriedade `align` com o valor `center`, **extremamente importante deixar um espaço entre o elemento e a imagem adicionar pelo Markdown, qualquer duvida acesse o readme do meu perfil

**Reply by PsyDrops** [reply]
Teria algum problema eu copiar o esqueleto do seu Readme?

**Reply by wesleyallan** [reply]
Nenhum problema, pode usar a vontade 👍
